cmd/kube-controller-manager/app/plugins.go

```diff
@@ -25,8 +25,6 @@ import (
 
 	"k8s.io/klog/v2"
 
-	// Cloud providers
-	cloudprovider "k8s.io/cloud-provider"
 	// ensure the cloud providers are installed
 	_ "k8s.io/kubernetes/pkg/cloudprovider/providers"
```

Comment:
is this needed?

Reply:
Yes, it's needed. It  populates the private variable providers map in the `k8s.io/cloud-provider` and is used to create the cloud provider instance for the given `--cloud-provider` flag in the kube-controller-manager. 

So we need keep it until the [KEP-2395](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cloud-provider/2395-removing-in-tree-cloud-providers) is GAed.